documentation/modules/cruise-control/con-optimization-proposals.adoc

```diff
@@ -94,10 +101,45 @@ m¦excludedBrokersForReplicaMove
 
 |===
 
+
+[discrete]
+=== Broker load
+
+The broker load section lists all broker IDs and provides a number of metrics for each broker. 
```

Comment:
Can we make it clear that the broker load is a json string in a config map. This reads like it is still a _section_ of the status message.

Reply:
It would also be nice to give an example command as to how a user might retrieve this load information json.